framework/tests/value/BitsMethodsIntRange.java

```diff
@@ -0,0 +1,15 @@
+public class BitsMethodsIntRange {
+    void caseInteger(int index, char val1, char val2) {
+        char[] arr1 = new char[33];
+        char[] arr2 = new char[33];
+        arr1[Integer.numberOfLeadingZeros(index)] = val1;
```

Comment:
I thought that the point of splitting this test was so that the value checker test does not use array indexing.
You can test that an expression is compatible with a qualified type by assigning to an annotate variable like here for example: https://github.com/typetools/checker-framework/blob/master/framework/tests/value/BitwiseAnd.java#L14

Reply:
@panacekcz I see now, the direct approach is to use `@IntRange()` directly, not using another part of the checker or array indexing, I was too hung up on the definition of `unit test` and `integration test`.